packages/protocol/test-sol/governance/network/BlockchainParameters.t.sol

```diff
@@ -8,11 +8,10 @@ import { Constants } from "@test-sol/constants.sol";
 import { Utils } from "@test-sol/utils.sol";
 
 contract BlockchainParametersTest is Test, Constants, Utils {
-  // using the mainnet epoch size would not allow to test an edge case
-  uint256 constant EPOCH_SIZE = 100;
```

Comment:
This should not be affected, since these test are running on forge. 

That being said, you might have to add some logic during integration tests that sets the `EPOCH_SIZE` back to 100, since all forge test currently have the EPOCH_SIZE set to 17280.

Reply:
That makes sense, thanks for clarifying